wgsl/index.bs

```diff
@@ -1229,7 +1229,13 @@ after the body of the nearest-enclosing [[#loop-statement]]
 or [[#switch-statement]].
 
 When a `break` statement is placed such that it would exit from a loop's [[#continuing-statement]],
-the break statement must appear last in that [[#continuing-statement]].
+then:
+
+* The `break` statement must appear as either:
+    * The only statement in the true-branch clause of an `if` that has no `else` clause and no `elseif` clauses, or
```

Comment:
Essentially, this says

* must be a break-if or break-unless that appears last in the continuing construct

Except that we just removed break-if and break-unless.